pkg/planner/agent.go

```diff
@@ -24,5 +25,5 @@ func DownloadClusterAgentYAML(ctx context.Context, url, ca string, token string)
 		return nil, err
 	}
 
-	return client.Get().AbsPath("v3", "import", token+".yaml").Do(ctx).Raw()
+	return client.Get().AbsPath("v3", "import", fmt.Sprintf("%v_%v.yaml", token, clusterID)).Do(ctx).Raw()
```

Comment:
Is this format dictated by upstream ? Or is it just anything we want it to be.

Reply:
This is dictated by Rancher: https://github.com/rancher/rancher/blob/master/pkg/multiclustermanager/routes.go#L82

Reply:
Well that makes it simple